src/site/stages/build/process-cms-exports/tests/transformed-entities/block_content-promo.json

```diff
@@ -11,20 +11,12 @@
             "value": "en"
           }
         ],
-        "bundle": [
```

Comment:
Now that we're omitting these properties during the filter step instead of skipping them during the entity expansion step, we've gotta remove them from the test data.

The only reason they were here in the first place is because an entity we _do_ have a transformer for (`block_content-promo`) has a reference to an entity we _don't_ have one for (`media-image`).